mirrord/agent/src/main.rs

```diff
@@ -555,14 +555,14 @@ async fn start_iptable_guard() -> Result<()> {
 
     let result = spawn_child_agent();
 
-    run_thread_in_namespace(
+    let _ = run_thread_in_namespace(
```

Comment:
Aren't we eating up the error now with this?

Reply:
It's by design since it can fail